### PR TITLE
Removing Validator Pages

### PR DESCRIPTION
**Pull request by samgutentag**

### CODE REVIEW

File: docs/api/blockchain/accounts.mdx
Comment (jthiller) [inline docs/api/blockchain/accounts.mdx:1]: We should honestly keep the references in the API. This has relevance for legacy API usage.

File: docs/api/blockchain/stats.mdx
Comment (jthiller) [inline docs/api/blockchain/stats.mdx:1]: Same here, lets not pull individual response content from old API docs.

File: docs/api/blockchain/validators.mdx
Comment (jthiller) [inline docs/api/blockchain/validators.mdx:1]: Same as part of old API

File: docs/blockchain/blockchain-primitives.mdx [hunk 1]
```diff
@@ -31,15 +31,7 @@ The easiest way to see the blocks being mined is with the
...
-### Epochs
-
-An `epoch` is the target time period for which a given group of Validators is elected to serve as
-the consensus group. The target time for an epoch is currently `30 blocks`, as defined in the
-`election_interval` chain variable. Approximately every 30 blocks mined marks the passing of an
-epoch, after which a new group of Validators is elected to form the next consensus group. Mining
-rewards are distributed per epoch (as opposed to per block in most blockchain-based systems). At the
-conclusion of each epoch, the consensus group will distribute all the $HNT produced in that block
-via the `rewards` transaction.
+<!--TODO: document epochs on soland and in helium rewards -->
```
Comment (jthiller) [inline docs/blockchain/blockchain-primitives.mdx]:
```suggestion
<!--TODO: document epochs on Solana and in helium rewards -->
```

File: docs/blockchain/blockchain-primitives.mdx [hunk 2]
```diff
@@ -31,15 +31,7 @@ The easiest way to see the blocks being mined is with the
...
 
 ### Transactions
```
Comment (jthiller) [inline docs/blockchain/blockchain-primitives.mdx]: This whole section could probably go. No more htlc, state channels, etc.

File: docs/blockchain/proof-of-coverage.mdx [hunk 3]
```diff
@@ -40,7 +40,7 @@ provided by Hotspots on the Network.
   [block 1,731,335](https://explorer.helium.com/blocks/1731335).
 - The first Oracled Proof of Coverage Reciepts were in
   [block 1,731,339](https://explorer.helium.com/blocks/1731339).
-- [Bitter Concrete Parakeet](https://explorer.helium.com/validators/12xtDibQdaAsvbPTqFbecGaA4iHd7NzRg6s7foQ2HCxouHo11TE)
+- [Bitter Concrete Parakeet](https://explorer-old.helium.com/validators/12xtDibQdaAsvbPTqFbecGaA4iHd7NzRg6s7foQ2HCxouHo11TE)
```
Comment (jthiller) [inline docs/blockchain/proof-of-coverage.mdx:43]: Should we do a sweep on all explorer links? Would need this for the blocks above too.

File: docs/home/home.mdx [hunk 4]
```diff
@@ -67,8 +67,8 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
         produce and are compensated in IOT, the native cryptocurrency of the Helium blockchain. The
         Helium blockchain is a new, open source, public blockchain created entirely to incentivize
         the creation of physical, decentralized wireless networks. Today, the Helium IoT Network,
-        and its <a href="https://explorer.helium.com/hotspots">hundreds of thousands of Hotspots</a>,
-        provide access to the largest LoRaWAN Network in the world.
+        and its <a href="https://explorer.helium.com/hotspots">hundreds of thousands of Hotspots</a>
```
Comment (jthiller) [inline docs/home/home.mdx]:
```suggestion
        and its <a href="https://explorer.helium.com/">hundreds of thousands of Hotspots</a>
```

File: docs/lorawan-on-helium/proof-of-coverage.mdx [hunk 5]
```diff
@@ -51,9 +51,9 @@ tabled because they were deemed too complex to implement will become viable and
 # Step 1: Oracle-Based PoC - In Progress
 
 From the launch of the network in 2019 until February 2, 2023, the IOT PoC system relied on a
-“Challenger, Beaconer, Witness, Validator, Rewarder” model. In this model at least five separate
-actors had to work together in order to make PoC events happen. Any break in the chain meant the
-loss of the entire event and its rewards.
+“Challenger, Beaconer, Witness, Rewarder” model. In this model at least five separate actors had to
```
Comment (jthiller) [inline docs/lorawan-on-helium/proof-of-coverage.mdx]: This paragraph is historical in context. Can revert this change.

File: docs/wallets/cli.mdx
Comment (jthiller) [inline docs/wallets/cli.mdx:1]: Fine messing with this doc since it has no continued utility.

File: netlify.toml [hunk 6]
```diff
@@ -93,6 +93,11 @@
   to = "/vote-escrow/voting-power"
   status = 301
   force = true
+[[redirects]]
+  from = "/validators/*"
+  to = "/solana/migration/validator-operator"
```
Comment (jthiller) [inline netlify.toml:98]: 👍

File: static/img/denylist/distributed_denylists_illustrated.png
Comment (jthiller) [inline static/img/denylist/distributed_denylists_illustrated.png:1]: 🥲

File: static/img/validators/faucet.png
Comment (jthiller) [inline static/img/validators/faucet.png:1]: Lets keep - will still have a faucet

**Review by jthiller — CHANGES_REQUESTED** [review]
SO MUCH CONTENT! 🔥

File: docs/api/blockchain/accounts.mdx
Comment (samgutentag) [reply docs/api/blockchain/accounts.mdx:1]: good call, restoring

File: docs/blockchain/blockchain-primitives.mdx (hunk 1 above)
Comment (samgutentag) [reply docs/blockchain/blockchain-primitives.mdx]: fixed!

File: docs/blockchain/blockchain-primitives.mdx (hunk 2 above)
Comment (samgutentag) [reply docs/blockchain/blockchain-primitives.mdx]: trimming

File: docs/lorawan-on-helium/proof-of-coverage.mdx (hunk 5 above)
Comment (samgutentag) [reply docs/lorawan-on-helium/proof-of-coverage.mdx]: reverted

**Review by jthiller — DISMISSED** [review]

**Review by jthiller — APPROVED** [review]

**Pull request merged**